peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -77,6 +77,17 @@ <h1 class="py-4">{{ document.title }}</h1>
                         {% endif %}
                       {% endblock %}
 
+                      {% block document-metadata-content-alternative-names %}
+                        <dt>{% trans 'Alternative names' %}</dt>
```

Comment:
Most documents won't have alternative names, so I think this should not show in the metadata block if there aren't any.